package.json

```diff
@@ -43,7 +43,7 @@
   },
   "scripts": {
     "start": "serve --single build/ --listen $PORT",
-    "dev": "craco start",
+    "dev": "REACT_APP_VERSION=$(git describe --tags --always) craco start",
```

Comment:
I purposely didn't introduce that change to keep it clean here.
I'd say simply run the "npm run dev" command with the git describe during your tests, or simply fill up the .env, but let's keep the package.json clear